spec/integration/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -11,4 +11,99 @@ spec:
     wait_for_data_timeout:
       description: Timeout for waiting for metrics data during a system test.
       type: string
-      example: 10m
\ No newline at end of file
+      example: 10m
+    agent:
+      description: "Configuration overrides for the Elastic Agent"
+      type: object
+      additionalProperties: false
+      properties:
+        runtime:
+          description: "Runtime to run the Elastic Agent process"
+          type: string
+          enum:
+            - docker
+        user:
+          description: "User that runs the Elastic Agent process"
+          type: string
+          example: root
+          default: ""
+        pid_mode:
+          description: >
+            (From docker-compose docs) Turns on sharing between container and the host
+            operating system the PID address space
+          type: string
+          enum:
+            - host
+        ports:
+          description: "List of ports to be exposed to access to the Elastic Agent"
+          type: array
+          items:
+            type: string
+            description: Port to be exposed, as defined in https://docs.docker.com/compose/compose-file/compose-file-v2/#ports
+            examples:
+              - "3000"
+              - "8000:8000"
+              - "127.0.0.1:7443:7443"
+              - "8000:8000/udp"
+        linux_capabilities:
+          description: "Linux Capabilities that must been enabled in the system to run the Elastic Agent process"
+          type: array
+          items:
+            type: string
+            description: Capability name
+            # https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities
+            enum:
```

Comment:
Do you think it should be added `ALL` here to this list too?

Reply:
We can add it later if needed, I think it would be better if developers need to check the capabilities the package really needs.